unit_tests/test_events.py

```diff
@@ -132,6 +132,23 @@ def test_filter_repair(self):
         self.assertIn("not filtered", log_content_after)
         self.assertNotIn("repair id 1", log_content_after)
 
+    def test_filter_upgrade(self):
```

Comment:
approved the PR but not sure why do we need this test, it looks the same as `test_filter_repair`